src/pages/ItemsList/components/FilterBar/components/FilterBarSize/filterBarSize.scss

```
@@ -0,0 +1,47 @@
+@import '/src/styles/variables.scss';
+@import '/src/pages/ItemsList/variablesForItemList.scss';
+@import url('https://fonts.googleapis.com/css2?family=Gothic+A1&display=swap');
+
+.filterSize {
+  border-top: 1px solid #e5e5e5;
+  padding-top: 10px;
+  margin-bottom: 15px;
+
+  .sizeHeader {
+    @include flexSort(space-between, center);
+    .sizeText {
+      padding-top: 7px;
+      font-weight: 1000;
+    }
+    .sizeIcon {
+      height: 14px;
+      margin-right: 9px;
+    }
+  }
+
+  .sizeNumbers {
+    display: flex;
+    flex-wrap: wrap;
+    margin-top: 17px;
+
+    .sizeNumber {
+      border: 1px solid #e5e5e5;
+      border-radius: 7px;
+      height: 37px;
+      width: 46px;
+      padding: 4px;
+      margin: 2px;
+      font-size: 12px;
+      @include flexCenter;
```

```suggestion
      @include flexCenter;
      border: 1px solid #e5e5e5;
      border-radius: 7px;
      height: 37px;
      width: 46px;
      padding: 4px;
      margin: 2px;
      font-size: 12px;
```
include는 선택자 최상단에 위치하는 것이 컨벤션입니다.

---

완료